vm/starnix/starnix.go

```diff
@@ -115,20 +126,33 @@ func (pool *Pool) Create(workdir string, index int) (vmimpl.Instance, error) {
 	if err := inst.setFuchsiaVersion(); err != nil {
 		return nil, fmt.Errorf(
 			"there is an error running ffx commands in the Fuchsia checkout (%q): %w",
-			inst.fuchsiaDirectory,
+			inst.fuchsiaDir,
 			err)
 	}
-	pubkey, err := osutil.RunCmd(30*time.Second, inst.fuchsiaDirectory, inst.ffxBinary, "config", "get", "ssh.pub")
+	pubkey, err := inst.runFfx(30*time.Second, "config", "get", "ssh.pub")
 	if err != nil {
 		return nil, err
 	}
 	inst.sshPubKey = string(bytes.Trim(pubkey, "\"\n"))
-	privkey, err := osutil.RunCmd(30*time.Second, inst.fuchsiaDirectory, inst.ffxBinary, "config", "get", "ssh.priv")
+	privkey, err := inst.runFfx(30*time.Second, "config", "get", "ssh.priv")
 	if err != nil {
 		return nil, err
 	}
 	inst.sshPrivKey = string(bytes.Trim(privkey, "\"\n"))
 
+	// Copy auto-detected product bundle path from in-tree ffx to isolated ffx.
```

Comment:
Good idea, glad this worked!

Reply:
I added some extra args/envvar settings to ensure that when running this non-isolated ffx call we don't write logs/settings anywhere, based on what we learned in your other PR.

Reply:
Marvelous!